Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -546,14 +568,14 @@ void doTeensyDebug() {
   ros_teensy_debug_pub.publish(&teensy_debug_msg);
 }
 
-/*
- * Converts a frequency in hertz to the
- * number of milliseconds to match the
- * frequency. For example, 2 hz = 500 millis
- * or 100 hz = 10 millis.
+/**
+ * Sets up a callback that is called at the
+ * given frequency.
  */
-int convertFreqToMillis(int frequency) {
-  return static_cast<int>((1.0/frequency) * 1000);
+void setupTimerCallback(int callback_freq_hz, void (*callback)(void)) {
```

Comment:
`setupTimerCallback` needs to be forward declared for `setup` to recognize it, according to travis.